templates/jitsi_meet_config.js.j2

```diff
@@ -86,9 +86,9 @@ var config = {
     // Audio
 
     // Disable measuring of audio levels.
-    // disableAudioLevels: false,
+    disableAudioLevels: {{ not ( jitsi_meet_show_audio_levels | bool ) | lower }},
```

Comment:
I am testing these changes now. `jitsi_meet_show_audio_levels` variable were not defined in *defaults/main.yml* yet.

Reply:
Better said, it wasn't renamed yet.

Reply:
Here it should be `{{ (not jitsi_meet_show_audio_levels) | bool | lower }}`, otherwise the result is `False` instead of `false` and config.json can't be parsed correctly

Reply:
> Better said, it wasn't renamed yet.

Sorry, I just saw that you had renamed this variable